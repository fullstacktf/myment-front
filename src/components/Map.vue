<template>
        <img src="../../public/assets/map.png" alt="map example">
</template>

<script>
export default {

}
</script>

<style scoped>

img{
margin: auto;
height: 400px;
width: 500px;
border: 1px solid black;
max-width: 100%;
}


</style>
