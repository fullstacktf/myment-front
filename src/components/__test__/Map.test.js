import Map from '../Map.vue';
import { shallowMount, createLocalVue } from '@vue/test-utils';
import Buefy from 'buefy';
import mock from '../../test/mockBack';

describe('Map', () => {
  let localVue;
  beforeEach(() => {
    const Vue = createLocalVue();
    Vue.use(Buefy);
    localVue = Vue;
  });
  test('render component', () => {
    const wrapper = shallowMount(Map, {
      localVue,
    });
    expect(wrapper.html()).toMatchSnapshot();
  });

  test('Check props', () => {
    
  });

  test('check Contents props', () => {
    
  });
});


