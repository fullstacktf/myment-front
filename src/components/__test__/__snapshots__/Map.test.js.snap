// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map render component 1`] = `"<img src=\\"../../public/assets/map.png\\" alt=\\"map example\\">"`;
